pkg/endpoints/cluster.go

```diff
@@ -162,3 +164,70 @@ func (r Resource) GetProperties(request *restful.Request, response *restful.Resp
 	properties := Properties{InstallNamespace: os.Getenv("INSTALLED_NAMESPACE")}
 	response.WriteEntity(properties)
 }
+
+// Get dashboard version
+func (r Resource) GetDashboardVersion(request *restful.Request, response *restful.Response) {
```

Comment:
No return type here - either return a nice response or the actual version as a string, and/or an error

Reply:
I have put errors in if cant get the deployment 
I have put errors in if it is unknown and it returns a 500 and the front end just doesn't display that field for now